build/common.props

```diff
@@ -55,6 +55,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
Supress "`Microsoft.Bcl.TimeProvider` doesn't support .NET Framework 4.6.1, consider upgrading [...]" messages. (Altough it's officially supported as a valid framework target).